tests/integration/cartography/intel/aws/test_s3.py

```diff
@@ -255,3 +255,86 @@ def test_s3_sns_relationship(neo4j_session):
     ) == {
         ("bucket-1", "arn:aws:sns:us-east-1:123456789012:test-topic"),
     }
+
+
+def test_load_s3_bucket_logging(neo4j_session):
+    """
+    Ensure that expected bucket gets loaded with their bucket logging fields.
```

Comment:
Nit: Please annotate with `# Arrange` + `# Act` + `# Assert` to make this clearer which parts are the things under test